vcd/datasource_vcd_nsxt_network_dhcp_binding.go

```diff
@@ -0,0 +1,139 @@
+package vcd
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func datasourceVcdNsxtDhcpBinding() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: datasourceVcdNsxtDhcpBindingRead,
+
+		Schema: map[string]*schema.Schema{
+			"org": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Description: "The name of organization to use, optional if defined at provider " +
+					"level. Useful when connected as sysadmin working across different organizations",
+			},
+			"org_network_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
```

Comment:
Same as `ForceNew` question in `org`

Reply:
This is the usual copy/paste flop :) I will remove it.